src/libraries/System.Formats.Tar/tests/TarTestsBase.cs

```diff
@@ -611,5 +612,22 @@ public static IEnumerable<object[]> GetPaxAndGnuTestCaseNames()
                 yield return new object[] { name };
             }
         }
+
+        public static void ExtractWithTarTool(string source, string destination)
+        {
+            Assert.True(!OperatingSystem.IsWindows(), "This method is unsupported on Windows.");
+
+            using Process tarToolProcess = new Process();
+            tarToolProcess.StartInfo.FileName = "/usr/bin/tar";
+            tarToolProcess.StartInfo.Arguments = $"-xf {source} -C {destination}";
+
+            tarToolProcess.StartInfo.UseShellExecute = false;
```

Comment:
This is the default.

Reply:
FWIW: I copied this options from `CreateSymbolicLink`:
https://github.com/dotnet/runtime/blob/9ea5d2d6cdc536fbe1dacc477ef2b2dea2b4845c/src/libraries/Common/tests/System/IO/ReparsePointUtilities.cs#L87-L98

Reply:
It's possible that was used at some point where it was also run on .NET Framework, where the default is true.